extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -768,7 +768,10 @@ export class AmpSlideScroll extends BaseSlides {
       );
       this.action_.trigger(this.element, name, event, opt_trust);
 
-      this.element.dispatchCustomEvent(name, {index: newIndex});
+      this.element.dispatchCustomEvent(name, {
+        index: newIndex,
+        ampTrust: actionTrustToString(opt_trust),
```

Comment:
Please add an issue to add this forward to 0.2.

Reply:
Done: #27082